### PR TITLE
Add dependencies required for tests

### PR DESCRIPTION
**Pull request by aaomidi**
These are required for the unit tests to run.

### CODE REVIEW

File: extensions/admin-tool-ext-win/package.json [hunk 1]
```diff
@@ -1,84 +1,86 @@
...
+      "admin-tool-ext-win"
+    ]
+  },
+  "dependencies": {
+    "ads-extension-telemetry": "github:Charles-Gagnon/ads-extension-telemetry#0.1.0",
+    "vscode-nls": "^3.2.1"
+  },
+  "devDependencies": {
+    "vscode": "^1.1.10",
+    "mocha-junit-reporter": "^1.17.0",
```
Comment (Charles-Gagnon) [inline extensions/admin-tool-ext-win/package.json]: I think the suggested way now is to use @types/vscode for typing and vscode-test for the actual test runner stuff

File: extensions/cms/package.json [hunk 2]
```diff
@@ -628,6 +628,8 @@
 	"devDependencies": {
 		"@types/mocha": "^5.2.5",
     "@types/node": "^10.14.8",
+    "mocha-junit-reporter": "^1.17.0",
```
Comment (Charles-Gagnon) [inline extensions/cms/package.json]: >     [](start = 0, length = 4)

spaces -> tabs

File: extensions/dacpac/package.json [hunk 3]
```diff
@@ -59,7 +59,8 @@
 		"@types/mocha": "^5.2.5",
 		"mocha": "^5.2.0",
 		"should": "^13.2.1",
-		"vscodetestcover": "^1.0.2"
+    "mocha-junit-reporter": "^1.17.0",
```
Comment (Charles-Gagnon) [inline extensions/dacpac/package.json]: >     [](start = 0, length = 4)

spaces -> tabs

File: extensions/dacpac/src/test/index.ts [hunk 4]
```diff
@@ -4,7 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 
 const path = require('path');
-import * as testRunner from 'vscodetestcover';
+const testRunner = require('vscode/lib/testrunner');
```
Comment (Charles-Gagnon) [inline extensions/dacpac/src/test/index.ts]: Why use require here?

File: extensions/dacpac/src/test/index.ts [hunk 5]
```diff
@@ -14,9 +14,9 @@ const testOptions: any = {
 	timeout: 60000
 };
 
-const coverageConfig: any = {
-	coverConfig: '../../coverageConfig.json'
-};
+// const coverageConfig: any = {
+// 	coverConfig: '../../coverageConfig.json'
+// };
```
Comment (Charles-Gagnon) [inline extensions/dacpac/src/test/index.ts]: Why removing?

File: extensions/resource-deployment/package.json [hunk 6]
```diff
@@ -364,6 +364,8 @@
   "devDependencies": {
     "typemoq": "^2.1.0",
     "vscode": "^1.1.26",
-    "@types/yamljs": "0.2.30"
+    "@types/yamljs": "0.2.30",
+    "mocha-junit-reporter": "^1.17.0",
+		"mocha-multi-reporters": "^1.1.7"
```
Comment (Charles-Gagnon) [inline extensions/resource-deployment/package.json]: >		 [](start = 0, length = 2)

Probably should format this whole file with tabs to match the others

File: extensions/schema-compare/package.json [hunk 7]
```diff
@@ -65,7 +65,9 @@
 	"devDependencies": {
 		"@types/mocha": "^5.2.5",
 		"@types/node": "^10.14.8",
-		"mocha": "^5.2.0",
+    "mocha": "^5.2.0",
+    "mocha-junit-reporter": "^1.17.0",
```
Comment (Charles-Gagnon) [inline extensions/schema-compare/package.json]: >     [](start = 0, length = 4)

spaces -> tabs

File: extensions/dacpac/src/test/index.ts [hunk 8]
```diff
@@ -29,6 +29,6 @@ if (process.env.BUILD_ARTIFACTSTAGINGDIRECTORY) {
 	};
 }
 
-testRunner.configure(testOptions, coverageConfig);
+testRunner.configure(options);
```
Comment (Charles-Gagnon) [inline extensions/dacpac/src/test/index.ts]: >options [](start = 21, length = 7)

This is breaking the build - there's no options var

File: extensions/admin-tool-ext-win/package.json (hunk 1 above)
Comment (aaomidi) [reply extensions/admin-tool-ext-win/package.json]: VSCode-test is great if you're running tests externally to the bits. They still use vscode for their own tests.

Comment (aaomidi) [reply extensions/admin-tool-ext-win/package.json]: It works a little differently and would require more changes than this to get it working. I'll leave it up to the test owners to update to that. ((It is necessary if we want mocha coverage reports))

File: extensions/dacpac/src/test/index.ts (hunk 4 above)
Comment (aaomidi) [reply extensions/dacpac/src/test/index.ts]: It's what all the other tests are using.

File: extensions/dacpac/src/test/index.ts (hunk 5 above)
Comment (aaomidi) [reply extensions/dacpac/src/test/index.ts]: `vscode/lib/testrunner` does not support coverage reports.

Comment (aaomidi) [reply extensions/dacpac/src/test/index.ts]: We decided that because of the test runner errors we rather get everything working then do test coverage.

Comment (Charles-Gagnon) [reply extensions/dacpac/src/test/index.ts]: Just delete this then until we figure out what we want to do here IMO

**Pull request merged**